anomalib/integration/__init__.py

```diff
@@ -0,0 +1,15 @@
+"""Integration."""
+
+# Copyright (C) 2021 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions
+# and limitations under the License.
```

Comment:
Rather than creating `anomalib/integration`, we could place `nncf_callback` and `utils.py` into `anomalib/utils/callbacks/nncf`. In this case, the existing flow would be kept as is. More specifically, the structure of `anomailb/utils/callback`would be something like the following:

```
.
├── __init__.py
├── callbacks
│   ├── __init__.py
│   ├── cdf_normalization.py
│   ├── compress.py
│   ├── min_max_normalization.py
│   ├── model_loader.py
│   ├── nncf
│   │   ├── __init__.py
│   │   ├── nncf_callback.py
│   │   └── utils.py
│   ├── save_to_csv.py
│   ├── tiler.py
│   ├── timer.py
│   └── visualizer_callback.py
```

Reply:
done